src/js/trove/world-lib.js

```diff
@@ -96,6 +142,23 @@ define(["js/runtime-util"], function(util) {
         clear_running_state();
     };
 
+    // Close all world computations.
+    Jsworld.shutdownSingle = function(options) {
+        if(runningBigBangs.length > 0) {
+            var currentRecord = runningBigBangs.pop();
+            currentRecord.pause();
+            if (options.cleanShutdown) {
+                currentRecord.success(world);
+            }
+            if (options.errorShutdown) {
+                currentRecord.fail(options.errorShutdown);
+            }
+        }
+        resume_running_state();
```

Comment:
I feel like the three lines below belong in `resume_running_state` as well, unless there are other planned uses for `resume_running_state` other than this call (this is the only call to it I see).  Since it has `resume` in the name, it seems like it should be the one to call `start`.